roles/openshift_health_checker/library/etcdstats.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/python
+# vim: expandtab:tabstop=4:shiftwidth=4
+'''
+Ansible module to determine the size of an etcd cluster up to a specified limit
+'''
+
+import etcd
+
+from ansible.module_utils.basic import AnsibleModule
+
+
+def get_size(client, key, total=0, limit=0, depth=0, depth_limit=1000, visited=None):
```

Comment:
So `depth=0, depth_limit=1000` is pretty pointless here.  We control the etcd keyspace, specifically we control exactly where objects are stored.  I wrote the most complicated logic for storing an object in etcd, which had a depth of `5`, so lets just pretend the absurd worst case is `10`.  Thus python will never even come close to a stackoverflow.  Therefore all this depth checking is just adding pointless complexity for no benefits.